packages/round-manager/src/features/common/FormDropDown.tsx

```diff
@@ -0,0 +1,129 @@
+import { FC } from "react";
+import {
+  Control,
+  useController,
+  UseFormRegisterReturn,
+  FieldErrors,
+  FieldPath,
+} from "react-hook-form";
+import { Listbox, Transition } from "@headlessui/react";
+import { CheckIcon, SelectorIcon } from "@heroicons/react/solid";
+import { Round } from "../api/types";
+import _ from "lodash";
+import { classNames } from "common";
+import { Fragment } from "react";
+
+export const FormDropDown: FC<{
+  register: UseFormRegisterReturn<string>;
+  errors: FieldErrors<Round>;
+  control: Control<Round>;
+  label: string;
+  id: FieldPath<Round>;
+  options: string[];
+  defaultValue: string;
+}> = ({ errors, control, label, id, options, defaultValue }) => {
+  // @ts-expect-error "required" appears on the error type, not sure where from
+  const errorMessage = _.get(errors, id)?.message;
+  const hasError = Boolean(errorMessage);
+  const { field } = useController({
+    name: id,
+    defaultValue: defaultValue,
+    control: control,
+    rules: {
+      required: true,
+    },
+  });
+
+  return (
+    <div className="col-span-6 sm:col-span-3 relative mt-2">
+      <Listbox {...field}>
+        {({ open }) => (
+          <div>
+            <Listbox.Label className="text-sm mt-4 mb-2">
+              <p className="text-sm">
+                <span>{label}</span>
+                <span className="text-right text-violet-400 float-right text-xs mt-1">
+                  *Required
+                </span>
+              </p>
+            </Listbox.Label>
+            <div className="mt-1 mb-2 shadow-sm block rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm">
+              <Listbox.Button
+                className={`relative w-full cursor-default rounded-md border h-10 bg-white py-2 pl-3 pr-10 text-left shadow-sm ${
+hasError
+? "border-red-300 text-red-900 placeholder-red-300 focus-within:outline-none focus-within:border-red-500 focus-within: ring-red-500"
+: "border-gray-300 focus:border-indigo-500 focus:outline-none focus:ring-1 focus:ring-indigo-500 sm:text-sm"
+}`}
+                data-testid={`${id}-testid`}
+                id={id}
+              >
+                <span className="flex items-center">
+                  {/*@ts-expect-error field.value includes the Date type, but we never actually render it */}
+                  <span className="ml-3 block truncate">{field.value}</span>
```

Comment:
Can we do a type check on this field and figure out what to render if it's not a string? Just in case future changes make the problem resurface.

Reply:
`{typeof field.value === 'string' ? field.value : field.value.toISOString()}` or thereabouts